packages/client/components/KudosReceivedNotification.tsx

```diff
@@ -25,15 +24,13 @@ const KudosReceivedNotification = (props: Props) => {
         picture
         meetingName
         meetingId
-        emoji
+        emojiUnicode
```

Comment:
@tianrunhe You have a little more context over the emoji issues than I do.  Did the original implementation only break in BQ?  Or were we also seeing issues in places like Amplitude?

Overall, since we're not yet adding the ability to customize the emoji, and we're likely going to revamp how we extract this into BQ early next year, I'm not too worried about this change.

Reply:
I currently store both: text representation and unicode. In this piece of code I just using `emojiUnicode` for rendering, text representation is still available.

Reply:
> Did the original implementation only break in BQ? Or were we also seeing issues in places like Amplitude?

That's right. Amplitude can display them fine. It's only BQ that's broken.